include/netutils/md5.h

```diff
@@ -74,24 +73,24 @@ extern "C"
  * Public Types
  ****************************************************************************/
 
-struct MD5Context
+struct md5_context_s
 {
   uint32_t buf[4];
   uint32_t bits[2];
   uint8_t in[64];
 };
 
-typedef struct MD5Context MD5_CTX;
+typedef struct md5_context_s MD5_CTX;
```

Comment:
The coding standard does not permit upper case letters in type name.  Types must always end in _t.  The correct name would be md5_ctx_t

Reply:
Hmm... there are other examples where upper case types are used for handles (after Microsoft conventions).  I suppose we should end that practice because it is non-compliant with the standard.

Reply:
Ok, I will fix line 13 and keep line 83 since you agree to merge without changing MD5_CTX.